plugins/pulp_rpm/plugins/importers/yum/importer.py

```diff
@@ -35,23 +35,30 @@ def metadata(cls):
             'types': [
                 ids.TYPE_ID_DISTRO, ids.TYPE_ID_DRPM, ids.TYPE_ID_ERRATA,
                 ids.TYPE_ID_PKG_GROUP, ids.TYPE_ID_PKG_CATEGORY, ids.TYPE_ID_RPM,
-                ids.TYPE_ID_SRPM, models.YumMetadataFile.TYPE, ids.TYPE_ID_PKG_ENVIRONMENT
+                ids.TYPE_ID_SRPM, ids.TYPE_ID_YUM_REPO_METADATA_FILE, ids.TYPE_ID_PKG_ENVIRONMENT
```

Comment:
+1 to using `models.<model_name>._content_type_id.default` instead of constants from `ids`. I've made this change for pre-merge.